.eslintrc.js

```diff
@@ -0,0 +1,257 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "commonjs": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "block-scoped-var": "error",
+        "block-spacing": [
+            "error",
+            "always"
+        ],
+        "brace-style": [
+            "error",
+            "stroustrup"
+        ],
+        "callback-return": "error",
+        "camelcase": "error",
+        "comma-spacing": "error",
+        "comma-dangle": [
```

Comment:
Nit: could we move this before `"comma-spacing"` so that it’s sorted alphabetically just like the other properties?